connectors/opa/lib/opa_reader.go

```diff
@@ -22,130 +22,195 @@ func NewOpaReader(opasrvurl string, client *http.Client) *OpaReader {
 	return &OpaReader{opaServerURL: opasrvurl, opaClient: client}
 }
 
-func (r *OpaReader) updatePolicyManagerRequestWithResourceInfo(in *taxonomymodels.PolicyManagerRequest, catalogMetadata map[string]interface{}) (*taxonomymodels.PolicyManagerRequest, error) {
-	responseBytes, errJSON := json.MarshalIndent(catalogMetadata, "", "\t")
+func (r *OpaReader) updatePolicyManagerRequestWithResourceInfo2(in *taxonomymodels.PolicyManagerRequest, catalogMetadata *datacatalogTaxonomyModels.DataCatalogResponse) (*taxonomymodels.PolicyManagerRequest, error) {
```

Comment:
Why 2 in updatePolicyManagerRequestWithResourceInfo2?

Reply:
fixed. Thanks.